lms/djangoapps/course_home_api/dates/v1/views.py

```diff
@@ -97,11 +96,8 @@ def get(self, request, *args, **kwargs):
         data = {
             'has_ended': course.has_ended(),
             'course_date_blocks': [block for block in blocks if not isinstance(block, TodaysDate)],
-            'missed_deadlines': missed_deadlines,
-            'missed_gated_content': missed_gated_content,
             'learner_is_full_access': learner_is_full_access,
             'user_timezone': user_timezone,
-            'verified_upgrade_link': verified_upgrade_deadline_link(request.user, course=course),
         }
```

Comment:
Some cleanup here in relation to the DatesBannerSerializerMixin, which is already providing these fields.

Reply:
We moved these fields to a different API? (One the MFE is already using?)

Reply:
They're still coming through this API, but from the `DatesBannerSerializerMixin` that's passed through the `DatesTabSerializer`.